config/coreboot-optiplex-7019_9010-maximized.config

```diff
@@ -568,23 +568,23 @@ CONFIG_CONSOLE_CBMEM=y
 # CONFIG_CONSOLE_SPI_FLASH is not set
 # CONFIG_CONSOLE_I2C_SMBUS is not set
 # CONFIG_DEFAULT_CONSOLE_LOGLEVEL_8 is not set
-CONFIG_DEFAULT_CONSOLE_LOGLEVEL_7=y
+# CONFIG_DEFAULT_CONSOLE_LOGLEVEL_7 is not set
 # CONFIG_DEFAULT_CONSOLE_LOGLEVEL_6 is not set
 # CONFIG_DEFAULT_CONSOLE_LOGLEVEL_5 is not set
 # CONFIG_DEFAULT_CONSOLE_LOGLEVEL_4 is not set
 # CONFIG_DEFAULT_CONSOLE_LOGLEVEL_3 is not set
 # CONFIG_DEFAULT_CONSOLE_LOGLEVEL_2 is not set
 # CONFIG_DEFAULT_CONSOLE_LOGLEVEL_1 is not set
-# CONFIG_DEFAULT_CONSOLE_LOGLEVEL_0 is not set
-CONFIG_DEFAULT_CONSOLE_LOGLEVEL=7
+CONFIG_DEFAULT_CONSOLE_LOGLEVEL_0=y
+CONFIG_DEFAULT_CONSOLE_LOGLEVEL=0
 CONFIG_CONSOLE_USE_LOGLEVEL_PREFIX=y
 CONFIG_CONSOLE_USE_ANSI_ESCAPES=y
 # CONFIG_CMOS_POST is not set
 CONFIG_POST_DEVICE_NONE=y
 # CONFIG_POST_DEVICE_LPC is not set
 # CONFIG_POST_DEVICE_PCI_PCIE is not set
 CONFIG_POST_IO_PORT=0x80
-CONFIG_HWBASE_DEBUG_CB=y
+CONFIG_HWBASE_DEBUG_NULL=y
```

Comment:
@miczyg1 also turns this on saving in oldconfig

Reply:
@miczyg1 Actually, if serial console output is the thing that slows things down (as discussed under #1806) then only serial console output should be deactivated. https://github.com/linuxboot/heads/compare/c5bbbe38fa8f6d4e5e92f7199dda7191245815a7..2424067203617b23f10ed48b65b8dea037e7b58b seems fine?

Reply:
Disabling serial console is fine as well.